packages/anasazi/src/AnasaziBlockKrylovSchur.hpp

```diff
@@ -1465,6 +1465,17 @@ namespace Anasazi {
           std::vector<int> bwork( curDim_ );
           int info = 0, sdim = 0; 
           char jobvs = 'V';
+
+          int opt_lwork = -1;
+          lapack.GEES( jobvs,curDim_, schurH_->values(), schurH_->stride(), &sdim, &tmp_rRitzValues[0],
+                       &tmp_iRitzValues[0], subQ.values(), subQ.stride(), &work[0], opt_lwork, 
+                       &rwork[0], &bwork[0], &info );
```

Comment:
For future reference: we can use e.g., `bwork.data()` instead of `&bwork[0]` now, since Trilinos only ever tests with C++11.